bin/scanpy-normalise-data.py

```diff
@@ -27,6 +30,10 @@ def main(args):
                            type=float,
                            default=1e4,
                            help='Aimed counts per cell after normalisation, default: 1e4')
+    argparser.add_argument('-r', '--save-raw',
```

Comment:
Why log scale? Conventionally (at least in R world), raw data is kept untransformed.

Reply:
It's log1p, so it's reversible. Anyway, this is what the scanpy tutorial is doing:

> Set the .raw attribute of AnnData object to the logarithmized raw gene expression for later use in differential testing and visualizations of gene expression. This simply freezes the state of the AnnData object returned by sc.pp.log1p.
> 
> `adata.raw = sc.pp.log1p(adata, copy=True)`

Reply:
Okay, fair enough.